app/controllers/cakes_controller.rb

```diff
@@ -1,4 +1,6 @@
 class CakesController < ApplicationController
+  helper ErrorsHelper
```

Comment:
I did it according to this: http://api.rubyonrails.org/classes/ActionController/Helpers.html 
I don't know if it is necessary or not.